_includes/tutorials/time-concepts/ksql/markup/dev/persistent-query.adoc

```diff
@@ -0,0 +1,5 @@
+Since the output looks right, the next step is to make the query persistent. This looks exactly like the push query, except we have added a `CREATE TABLE AS` statement to the beginning of it. This statement returns to the CLI prompt right away, having created a persistent stream processing program running in the ksqlDB engine, continuously processing input records and updating the resulting `MOVIE_TICKETS_SOLD` table. Moreover, we don’t see the results of the query displayed in the CLI, because they are updating the newly-created table itself. That table is available to other ksqlDB queries for further processing, and by default all its records are produced to a topic having the same name (MOVIE_TICKETS_SOLD).
```

Comment:
This file isn't used in the harness yaml for time-concepts. Can we remove this?

Reply:
@awalther28 good catch.  Removing...